### PR TITLE
WIP: Customizations for plssph3

### PR DESCRIPTION
**Pull request by ariedel**
The following changes have been made for getting plssph3 to work:
* The password file location has been updated
* The database has been switched to an actual database with data that can validate logins
* The hard-coded list of URLs has been updated with the new test slots (and broken out into a nice user-readable json file)
* Some Py26-specific code is no longer needed, now that we're running on py27

### CODE REVIEW

File: ssph_server/db.py [hunk 1]
```diff
@@ -54,10 +54,10 @@
 if 1 :
     import pandokia.db_mysqldb as d
     core_db = d.PandokiaDB( {
-            'host'      : 'tlssphdbv1', # plssphv1 for production
+            'host'      : 'plssphdbv1',
             'port'      : 3306,
             'user'      : 'etcadmin',
-            'passwd'    : password, # stored in /usr/lib/python2.6/site-packages/ssph_server/pswd
+            'passwd'    : password,
```
Comment (cslocum) [inline ssph_server/db.py]: I think we should keep this comment, with the path updated.

File: ssph_server/db.py [hunk 2]
```diff
@@ -54,10 +54,10 @@
 if 1 :
     import pandokia.db_mysqldb as d
     core_db = d.PandokiaDB( {
-            'host'      : 'tlssphdbv1', # plssphv1 for production
+            'host'      : 'plssphdbv1',
```
Comment (cslocum) [inline ssph_server/db.py]: Can you remind me why this change was made?

File: ssph_server/urllist.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "test1": "52.4.225.70/32",
+    "test1_main": "52.207.161.178/32",
+    "test6_main": "34.201.96.92/32",
+    "test2_main": "52.206.239.246/32",
+    "test3_main": "34.192.25.186/32",
+    "test4_main": "34.192.26.162/32",
+    "test5_main": "34.192.25.137/32",
+    "test7_main": "52.21.154.45/32",
+    "test8_main": "52.54.73.201/32"
+}
```
Comment (cslocum) [inline ssph_server/urllist.json]: :tada:

File: ssph_server/db.py (hunk 2 above)
Comment (ariedel) [reply ssph_server/db.py]: We couldn't authenticate with tlssphdbv1... although, that may have been for other reasons.

**Comment by ariedel** [comment]
Now that the SSPH servers work, I have removed most of the debugging lines.
